templates/page_coc.html

```diff
@@ -0,0 +1,243 @@
+          Don’t publish individual photographs of people without their consent.
+        </li>
+        <li>
+          Be careful in the words that you choose. Remember that sexist, racist,
+          and other exclusionary jokes can be offensive and unwelcoming to those
+          around you.
+        </li>
+        <li>
+          Excessive swearing and offensive jokes are not appropriate for
+          DjangoCon Africa.
+        </li>
+      </ul>
+    </div>
+
+    <div>
+      <h2 class="mb-4">Where does the Code of Conduct apply?</h2>
+      <p>
+        This Code of Conduct applies to all conference related spaces. That
+        includes, but is not limited to:
+      </p>
+      <ul>
+        <li>The conference venue(s)</li>
+        <li>The conference hotel(s)</li>
+        <li>Any conference related social activities</li>
+        <li>
+          Slack channels, tweets with the conference hashtag, and other online
+          media
+        </li>
+      </ul>
+      <br />
+      <p>
+        The Code of Conduct does not exclusively apply to events on the
+        conference agenda. For example, if after a scheduled social event you go
+        to a bar with a group of fellow participants, and someone harasses you
+        there, we would still treat that as a CoC violation.
+      </p>
+    </div>
+
+    <div>
+      <h2 class="mb-4">What can happen if the CoC is violated?</h2>
+      <p>
+        In case of a Code of Conduct violation, some of the most common actions
+        organisers may take are:
+      </p>
+      <ul>
+        <li>Demanding that a participant stops their behavior.</li>
+        <li>
+          Demanding that a participant prevents further contact with certain
+          other participants.
+        </li>
+        <li>Not publishing the video of a conference talk.</li>
+        <li>Cancelling a conference talk.</li>
+        <li>Removing a participant from the conference, without refund.</li>
+      </ul>
+      <br />
+      <p>
+        The action taken is at the discretion of the Code of Conduct team.
+        Participants are expected to comply immediately, and further action may
+        be taken in case a participant does not comply. A record will be kept of
+        all incidents.
+      </p>
+    </div>
+
+    <div>
+      <h2 class="mb-4">Where to report incidents</h2>
+      <p>
+        If a Code of Conduct incident happens to you, or witness it happening to
+        someone else, please contact the CoC team immediately either in person,
+        or by sending an email to <a
+          href="mailto:conduct@djangocon.africa"
+          class="underline inline-block"
+          >conduct@djangocon.africa</a
+        >. In urgent cases, you can also phone us on. <br />
+        Our Code of Conduct team consists of:
+      </p>
+      <ul>
+        <li>Anna Makarudze</li>
+        <li>Daniele Procida</li>
+        <li>Jessica Upani</li>
+        <li>Humphrey Butau</li>
+        <li>Abigail Dogbe</li>
```

Comment:
Could you kindly remove her, she is not part of this year's organising committee.